frontend/src/components/results/PlayerResultsItem.tsx

```diff
@@ -0,0 +1,156 @@
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.12);
+  
+  td {
+    background-color: ${({ theme }) => theme.colors.white};
+  }
+`;
+
+const PlayerContent = styled.td`
+  height: 40px;
+  line-height: 40px;
+  text-align: left;
+`;
+
+const PlaceColumn = styled.td`
+  width: 70px;
+`;
+
+const CodeColumn = styled.td`
+  width: 140px;
+`;
+
+const CircleIcon = styled.div<CircleParams>`
+  display: inline-block;
+  border-radius: 50%;
+  background: ${({ theme, color }) => theme.colors.gradients[color]};
+  margin: 0 10px 0 0;
+  width: 24px;
+  height: 24px;
+`;
+
+const PreviewContainer = styled.div`
+  display: flex;
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+`;
+
+const PreviewIcon = styled.i`
+  font-size: ${({ theme }) => theme.fontSize.default};
+  margin-left: 5px;
+`;
+
+const PlayerText = styled(LowMarginText)`
+  display: inline-block;
+`;
+
+const FlexCenter = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: left;
+`;
+
+type CircleParams = {
+  color: string,
+};
+
+type PlayerResultsCardProps = {
+  player: Player,
+  place: number,
+  isCurrentPlayer: boolean,
+  gameStartTime: string,
+  color: Color,
+  onViewCode: () => void,
+};
+
+function PlayerResultsItem(props: PlayerResultsCardProps) {
+  const {
+    player, place, isCurrentPlayer, color, gameStartTime, onViewCode,
+  } = props;
+
+  const bestSubmission = useBestSubmission(player);
+
+  const getDisplayNickname = () => {
+    const { nickname } = player.user;
+    return `${nickname} ${isCurrentPlayer ? '(you)' : ''}`;
+  };
+
+  const getScore = () => {
+    if (!bestSubmission) {
+      return '0';
+    }
+
+    const percent = Math.round((bestSubmission.numCorrect / bestSubmission.numTestCases) * 100);
+    return `${percent}%`;
+  };
+
+  const getSubmissionTime = () => {
+    if (!bestSubmission) {
+      return 'N/A';
+    }
+
+    // Calculate time from start of game till best submission
+    const startTime = new Date(gameStartTime).getTime();
+    const diffMilliseconds = new Date(bestSubmission.startTime).getTime() - startTime;
+    const diffMinutes = Math.floor(diffMilliseconds / (60 * 1000));
+
+    return ` ${diffMinutes} min`;
+  };
+
+  const getSubmissionCount = () => player.submissions.length || '0';
+
+  const getSubmissionLanguage = () => {
+    if (!bestSubmission) {
+      return 'N/A';
+    }
+
+    return (
+      <PreviewContainer>
+        <TextButton onClick={onViewCode}>
+          {displayNameFromLanguage(bestSubmission.language as Language)}
+          <PreviewIcon className="material-icons">launch</PreviewIcon>
+        </TextButton>
+      </PreviewContainer>
+    );
+  };
+
+  return (
+    <Content>
+      <PlaceColumn>
+        <PlayerText bold>{`${place}. `}</PlayerText>
+      </PlaceColumn>
+
+      <PlayerContent>
+        <FlexCenter>
+          <CircleIcon color={color.gradientColor} />
+          <PlayerText bold={isCurrentPlayer}>{getDisplayNickname()}</PlayerText>
```

Comment:
Cap the nickname size using the same system used on the Game page (see image).

<img width="1276" alt="Results Page Nickname Size Issue" src="https://user-images.githubusercontent.com/7987279/112767007-3f695c00-8fc9-11eb-8c42-763b53baa5e6.png">

Reply:
Good catch, done! Will wait for the max questions database issue to resolve before committing though that way I can actually test that it works :/

Reply:
Here's how it looks now:

<img width="519" alt="Podium with max width and scroll" src="https://user-images.githubusercontent.com/24768574/112796746-415e0a00-901f-11eb-8dad-e8247b94e09b.png">